schedule/yast/maintenance/yast-mru-install-minimal-with-addons_sle15sp3_dev_x86.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
`_dev.yaml` would be more visible at first view, so we can distinguish quickly

Reply:
changed.